tests/views/test_reporting_units.py

```diff
@@ -0,0 +1,40 @@
+import json
+import unittest
+
+import requests_mock
+
+from config import TestingConfig
+from response_operations_ui import app
+
+
+url_get_reporting_unit = f'{app.config["BACKSTAGE_API_URL"]}/v1/reporting-unit/50012345678'
+with open('tests/test_data/reporting_units/reporting_unit.json') as json_data:
+    reporting_unit = json.load(json_data)
+
+
+class TestReportingUnits(unittest.TestCase):
+
+    def setUp(self):
+        app_config = TestingConfig()
+        app.config.from_object(app_config)
```

Comment:
Why are you specifying here to use the TestingConfig when this will only be run in the test suite already using TestingConfig?

Reply:
If we want to run individual tests then we'd have to manually set `APP_SETTINGS=TestingConfig` with environment variables. I think this removes this requirement?

Reply:
This will allow the tests to be run individually/by case without using the run_tests.py script

On a similar note I think we could maybe get rid of that script in favour of using a Makefile

Reply:
I agree with that ^